server/controller/user.js

```diff
@@ -8,14 +8,14 @@ dotenv.config();
 const secret = process.env.SECRET;
 class UserController{
     static signUp (req, res, next) {
-        const query = 'INSERT INTO users(firstname, lastname, about, email, password, confirmpassword, created_at) VALUES($1, $2, $3, $4, $5, $6, Now() ) RETURNING *';
+        var email = req.body.email.split(' ').join('')
+        const query = 'INSERT INTO users(firstname, lastname, about, email, password, created_at) VALUES($1, $2, $3, $4, $5, Now() ) RETURNING *';
         const values =[
             req.body.firstname, 
             req.body.lastname, 
             req.body.about, 
-            req.body.email, 
+            email, 
             req.body.password,
-            req.body.confirmPassword
         ]
         client.query(query, values, (err, data) => {
             if (err) return res.status(409).send({
```

Comment:
Nothing happens if there is error? and also why is the status code 409?

Reply:
409 is for conflict if user with the same email details already exist.

Reply:
But we would also get 409 for every err here because you don't just check if the email address exists, you checked if there is err return 409 which is wrong

Reply:
But we would also get 409 for every err here because you don't just check if the email address exists, you checked if there is err return 409 which is wrong.